enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -3,18 +3,23 @@
 """
 import logging
 
+from django.db.models import Case, Exists, F, Max, OuterRef, Prefetch, Q, Value, When
```

Comment:
I think this one line exemplifies my concerns over the last week about frontend-backend design compatibility and tech debt.  There's a time and place for ORM magic, this should not have been one of them.